### DISCUSSION TITLE
Delete Individual Address Bar Search Suggestions

### DISCUSSION THREAD

**Discussion by ZoinktheBoink** [Floorp-Projects/Floorp#849] [Feature Request]
Could you please add the function to delete individual search suggestions from the address bar with `shift + delete`?

Info: https://support.mozilla.org/en-US/kb/remove-websites-address-bar-suggestions#w_clear-a-single-item-from-the-address-bar-history

**Comment by Telmesomething** [comment]
Type "about:config" (without quotes) in the URL bar and hit Enter. Confirm you'll be careful is the warning appears. Where it says **Search Prerence Name** the top, copy/paste **browser.urlbar.suggest.searches** and set it to **false** You can do that either by double clicking it, or by using the Toggle on the right hand side. 

No need to save anything, or reboot. It works straight away and remains disabled until you reverse the procedure by setting it to **true**

Other options you might be interested in on the same subject are these:

- **browser.urlbar.maxRichResults** and set it to 4
- **browser.urlbar.oneOffSearches** and set it to _false_

The first one reduces the number of entries which appear to 3 (plus the first one which counts as 1).
The second one looks like it's been deprecated but might be reinstated (hopefully). Either way, you can toggle it back on easily enough.
